extensions/blocks/send-a-message/edit.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * External dependencies
+ */
+import { InnerBlocks } from '@wordpress/block-editor';
+
+export default function SendAMessageEdit( props ) {
+	const { className } = props;
```

Comment:
We could also destructure in the function param: `function SendAMessageEdit( { className } )`